### DISCUSSION TITLE
Tasmota - load power cycle

### DISCUSSION THREAD

**Discussion by sewilson1914** [arendst/Tasmota#19551] [Q&A] [answered]
Let me say first that I'm new to this - just learning.  I have a Kaufmann PLF12 that I've flashed Tasmota onto.  

I need to setup a smart plug that can power off the load – wait 30 seconds – and power it back on.  However, the load is the internet connection, so the power-on command has to be embedded in the device; the command cannot come from the internet.

I also need to setup something very simple for the end-user that will allow them to run the power cycle routine from their iOS device.

Can you point me in the right direction?

**Comment by lalo-uy** [comment]
With pulsetime you can setup the time the relay is on.
Then seting it as inverted relay (Relai) in the configuration. The on and off are inverted.
Using the Web UI with renamed buttons could be enough , YMMV

**Reply by sfromis** [reply, marked as answer]
Instead of inverted relay, I'd suggest PowerOnState 5, for "inverted PulseTime". Then it will turn off for PulseTime, and then automatically on again. Avoids messing around with changing text for buttons and JSON payloads.
